mopidy_musicbox_webclient/static/js/gui.js

```diff
@@ -316,7 +320,8 @@ function initSocketevents () {
     })
 
     mopidy.on('event:streamTitleChanged', function (data) {
-        setSongTitle(data.title, true)
+        // Update all track info.
+        mopidy.playback.getCurrentTlTrack().then(processCurrenttrack, console.error)
```

Comment:
Out of interest, is this a bug fix or is this to make the popup also change, or something?